core/src/test/java/org/opensearch/sql/expression/ReferenceExpressionTest.java

```diff
@@ -138,58 +137,33 @@ public void array_with_multiple_path_value() {
   }
 
   /**
-   * {
-   *   "name": "bob smith"
-   *   "project.year": 1990,
-   *   "project": {
-   *     "year": 2020
-   *   },
-   *   "address": {
-   *     "state": "WA",
-   *     "city": "seattle"
-   *     "project.year": 1990
-   *   },
-   *   "address.local": {
-   *     "state": "WA",
-   *   },
-   *   "message": [
-   *     { "info": "message in array" },
-   *     { "info": "Only first index of array used" }
-   *   ]
-   * }
+   * { "name": "bob smith" "project.year": 1990, "project": { "year": 2020 }, "address": { "state":
+   * "WA", "city": "seattle" "project.year": 1990 }, "address.local": { "state": "WA", }, "message":
+   * [ { "info": "message in array" }, { "info": "Only first index of array used" } ] }
```

Comment:
Looks like this lost the nice formatting, I think it might need the <pre> tag

Reply:
Good catch, thanks! Fixed.